test/common/api/bpdIoLogin.js

```diff
@@ -11,10 +11,10 @@ export function login(baseUrl, fiscalCode) {
     return res.body
 }
 
-export function loginFullUrl(url, fiscalCode) {
+export function loginFullUrl(url, fiscalCode, subscriptionKey) {
```

Comment:
revert this changes! as suggested in
https://github.com/pagopa/cstar-integration-test/pull/80/files

you should not more invoke this function